artcommon/artcommonlib/util.py

```diff
@@ -430,3 +430,15 @@ def detect_package_managers(metadata, dest_dir: Path):
         if any(dest_dir.joinpath(file).is_file() for file in files):
             pkg_managers.append(pkg_manager)
     return pkg_managers
+
+
+@retry(reraise=True, wait=wait_fixed(10), stop=stop_after_attempt(3))
+async def get_konflux_slsa_attestation(pull_spec: str):
+    """
+    Inspect the SLSA provenance: https://konflux.pages.redhat.com/docs/users/metadata/attestations.html
+    To see if the latest build is actually hermetic
```

Comment:
nit: think this comment can be removed, since we're just returning the attestation and func caller can decide what to do with it

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1570/commits/e7f988d9c6288eea1bebb7772a03797f100cead3